teamcity.sh

```diff
@@ -1,18 +1,14 @@
 #!/bin/bash
 
-target=target/packages/memb-eventbriteImages
-base_files="bin config node_modules prod-config server.sh package.json"
-app_files="config controllers public service views routes.js server.js"
+####################################################################################################################
+#### Turned off because a load of unnecessary babel dependencies get installled and slow down deployments ##########
```

Comment:
This could be vulnerable to a node version change on the build agent. Might be worth explicitly naming the node version.